installer/install.iss

```diff
@@ -2525,9 +2553,21 @@ begin
     DeleteMarkedEnvString('GIT_SSH');
     DeleteMarkedEnvString('SVN_SSH');
 
-    if (PuTTYPage<>NIL) and RdbSSH[GS_Plink].Checked then begin
-        SetAndMarkEnvString('GIT_SSH',EdtPlink.Text,True);
-        SetAndMarkEnvString('SVN_SSH',EdtPlink.Text,True);
+    if (PuTTYPage<>NIL) then begin
+        if RdbSSH[GS_Plink].Checked then begin
+            SetAndMarkEnvString('GIT_SSH',EdtPlink.Text,True);
+            SetAndMarkEnvString('SVN_SSH',EdtPlink.Text,True);
+            if TortoisePlink.Checked then begin
+                Log('Line {#__LINE__}: Setting config option ssh.variant=tortoiseplink');
+                GitSystemConfigSet('ssh.variant','tortoiseplink')
+            end else begin
+                Log('Line {#__LINE__}: Setting config option ssh.variant=putty');
+                GitSystemConfigSet('ssh.variant','putty');
+            end;
+        end else begin
+            Log('Line {#__LINE__}: Clearing config option ssh.variant');
+            GitSystemConfigSet('ssh.variant',#0);
```

Comment:
Since this is the much shorter conditional arm of the `if ... then ... else` construct, maybe negate the condition and switch the arms around? That way, it would be easier to read.

Reply:
Ok.